src/compas/geometry/primitives/shapes/cone.py

```diff
@@ -167,6 +171,35 @@ def volume(self):
         """Float: The volume of the cone."""
         return pi * self.circle.radius**2 * (self.height / 3)
 
+    def to_vertices_and_faces(self, **kwargs):
+        """Returns a list of vertices and faces, called by `Mesh.from_shape()`."""
+        if 'u' in kwargs:
```

Comment:
you could simplify these defaults with one line:
```
    u = kwargs.get('u', 10)
```

Reply:
great suggestion! will do